windows/scripts/BinplaceCudaDependencies.cmd

```diff
@@ -0,0 +1,22 @@
+set CUDA_TOOLKIT_BIN_DIR=%~1%
+set CUDNN_PATH=%~2%
+set IS_CPU_ONLY_BUILD=%3%
+set USE_CUDNN=%4%
+set OUTPUT_DIR=%~5%
+
+if %IS_CPU_ONLY_BUILD% == true (
+    echo BinplaceCudaDependencies : CPU only build, don't copy cuda dependencies.
+ ) else (
+    echo BinplaceCudaDependencies : Copy cudart*.dll, cublas*dll, curand*.dll to output.
```

Comment:
You don't have to install CUDA toolkit and have CUDA in your PATH on machine where you run trainings. Its enough to install CUDA driver, and with necessary CUDA dlls in output it's enough to run trainings.